<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectForm</class>
 <widget class="QWidget" name="selectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JESD</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Список устройств:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="devListComboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="refrDevListPushButton">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>refresh.png</normaloff>refresh.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="LEDDevPushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Мигнуть светодиодом</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectDevPushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>84</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Подключиться</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
